### PR TITLE
adding in docker image content signing

### PR DESCRIPTION
**Pull request by joshuafernandes**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -493,15 +493,30 @@ task dockerUpload {
...
       }
 
       exec {
         workingDir dockerBuildDir
         executable "sh"
-        args "-c", "docker buildx build --platform ${dockerPlatforms} --build-arg BUILD_DATE=${buildTime()} --build-arg VERSION=${dockerBuildVersion} --build-arg VCS_REF=${getCheckedOutGitCommitHash()} --push ${tags} ."
+        args "-c", "docker buildx build --platform ${dockerPlatforms} --build-arg BUILD_DATE=${buildTime()} --build-arg VERSION=${dockerBuildVersion} --build-arg VCS_REF=${getCheckedOutGitCommitHash()} ${tags} ."
+      }
+
+      for (def tag in dctTags) {
```
Comment (joshuafernandes) [inline build.gradle:514]: DCT signing is one tag at a time only as of now

**Review by usmansaleem — APPROVED** [review]
LGTM

**Review by Ezzahhh — APPROVED** [review]
LGTM!

File: build.gradle [hunk 2]
```diff
@@ -493,15 +493,30 @@ task dockerUpload {
...
+      def dctTags = []
+      versionPrefixes.forEach { prefix ->
+        tags += "-t ${dockerImageName}:${prefix.trim()}-${variant} "
+        dctTags.add("${dockerImageName}:${prefix.trim()}-${variant}")
+      }
       if (variant == dockerVariants[0]) {
-        versionPrefixes.forEach { prefix -> tags += "-t ${dockerImageName}:${prefix.trim()} "}
+        versionPrefixes.forEach { prefix ->
+          tags += "-t ${dockerImageName}:${prefix.trim()} "
+          dctTags.add("${dockerImageName}:${prefix.trim()}-${variant}")
```
Comment (usmansaleem) [inline build.gradle:504]: we don't need variant here

**Review by jframe — APPROVED** [review]
LGTM

**Review by usmansaleem — APPROVED** [review]
LGTM

File: build.gradle (hunk 2 above)
Comment (joshuafernandes) [reply build.gradle]: nice catch, fixed

**Pull request merged**